code/src/Thought.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import moment from 'moment'
+import { LikeButton } from './LikeButton'
+
+export const Thought = (props) => (
```

Comment:
In case you didn't know - you don't need `()` around when you are only passing one argument
```suggestion
export const Thought = props => (
```